src/components/common/EventDialog.js

```diff
@@ -4,9 +4,10 @@ import { View } from 'react-native'
 import { Portal, Dialog, Paragraph, Text } from 'react-native-paper'
 import CustomButton from './CustomButton'
 import type { TransactionEvent } from '../../lib/gundb/UserStorage'
-import Avatar from './Avatar'
 import Section from './Section'
 import { BigGoodDollar } from '../common'
+import Avatar from './Avatar'
```

Comment:
can we import `Avatar` from `../common`?

Reply:
Done